--- lginst/version.cc.orig	2005-07-24 13:54:23.000000000 +0200
+++ lginst/version.cc	2005-07-24 13:55:06.000000000 +0200
@@ -72,12 +72,12 @@
 char* LIFEGEEK_VERSION = "1.03stable";
 
 #if !defined CONFIG_DOS
-char* MAINFILE = "/usr/local/share/lifegeek/main.lg";
-char* LICENSEFILE = "/usr/local/share/lifegeek/license.lg";
-char* WASTEFILE = "/usr/local/share/lifegeek/waste.lg";
-char* JOBFILE = "/usr/local/share/lifegeek/job.lg";
-char* SCOREFILE = "/usr/local/share/lifegeek/scores.lgs";
-char* SAVEFILES = "/usr/local/share/lifegeek/";
+char* MAINFILE = "/usr/share/lifegeek/main.lg";
+char* LICENSEFILE = "/usr/share/lifegeek/license.lg";
+char* WASTEFILE = "/usr/share/lifegeek/waste.lg";
+char* JOBFILE = "/usr/share/lifegeek/job.lg";
+char* SCOREFILE = "/var/games/lifegeek/scores.lgs";
+char* SAVEFILES = "/var/games/lifegeek/";
 #else
 char* MAINFILE = "main.lg";
 char* LICENSEFILE = "license.lg";
