diff -burN lifegeek.orig/version.c lifegeek/version.c
--- lifegeek.orig/version.c	2004-04-02 03:50:16.000000000 +0200
+++ lifegeek/version.c	2004-04-04 13:15:32.914395160 +0200
@@ -61,10 +61,10 @@
 
    NOTE: Before you change the install locations, see the file INSTALL for
    important information. */
-char* MAINFILE = "/usr/local/share/lifegeek/main.lg";
-char* LICENSEFILE = "/usr/local/share/lifegeek/license.lg";
-char* WASTEFILE = "/usr/local/share/lifegeek/waste.lg";
-char* JOBFILE="/usr/local/share/lifegeek/job.lg";
+char* MAINFILE = "/usr/share/lifegeek/main.lg";
+char* LICENSEFILE = "/usr/share/lifegeek/license.lg";
+char* WASTEFILE = "/usr/share/lifegeek/waste.lg";
+char* JOBFILE="/usr/share/lifegeek/job.lg";
 
 /* Score file
 
@@ -74,11 +74,11 @@
    NOTE: Before you change the install locations, see the file INSTALL for
    important information. */
 
-char* SCOREFILE = "/usr/local/share/lifegeek/scores.lgs";
+char* SCOREFILE = "/var/games/lifegeek/scores.lgs";
 
 /* Save files in this directory
    NOTE: Before you change the install locations, see the file INSTALL for
    important information. */
-char* SAVEFILES = "/usr/local/share/lifegeek";
+char* SAVEFILES = "/var/games/lifegeek";
 
 /* There are additional settings in version.h that you need to define as well */
